src/pages/dashboard/charts/_Map.js

```diff
@@ -19,8 +19,8 @@ const margin = {
   right: 10,
   top: 10,
 }
-const height = 400
-const width = 700
+const height = 520
```

Comment:
This totally works, thanks! I wonder if we should maybe do something like get the width of the document and then make the width related to that? Let's merge this for now and reconsider that later